web/packages/teleterm/package.json

```diff
@@ -35,7 +35,7 @@
     "winston": "^3.3.3"
   },
   "devDependencies": {
-    "@electron/notarize": "^2.4.0",
+    "@electron/notarize": "^3.0.1",
```

Comment:
I have to drop this to 2.5.0. 3.0.0 removed support for Node.js v20 and specifiecs `engines.node`. Yarn does a hard fail if `engines.node` doesn't match what's on the system.  Our CI is still on v20, so this won't go through until we add `--ignore-engines` to `yarn install`.

Instead of doing that, I'll just downgrade this package. It should still work and Connect v15 is going to be EOL soon anyway.

Reply:
Makes sense, let's downgrade the package.